### PR TITLE
Migrate to rails 4.0.x

### PR DESCRIPTION
**Pull request by lucianosousa**

### CODE REVIEW

**Comment by lucianosousa** [comment]
@ricardovaleriano done! ;)

**Comment by ricardovaleriano** [comment]
WOW! :clap: :clap: :clap: :clap:
Looking good man.
Thank you for this!

I will take a look as soon as possible ok?
Thank you very much.

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
   # The secret key used by Devise. Devise uses this key to generate
   # random tokens. Changing this key will render invalid all existing
   # confirmation, reset password and unlock tokens in the database.
-  config.secret_key = ENV['SECRET_KEY']
+  config.secret_key = '6525b61f9ea5fe6f8f037be5c2fc00294ddc3f46c54626bd9b39c52760134207106c10805cd06135fa0306b7321a11e44dfd08e622954e75d1fb9f7bed659b1a'
```
Comment (rodrigopinto) [inline config/initializers/devise.rb]: @lucianosousa It probably should keep use the ENV variable.

Comment (ricardovaleriano) [inline config/initializers/devise.rb]: @rodrigopinto :bomb: No doubt! =)

Comment (rodrigopinto) [inline config/initializers/devise.rb]: @ricardovaleriano I've took a look and everything looks good, I think just need this update.

Comment (ricardovaleriano) [inline config/initializers/devise.rb]: Great @rodrigopinto! Thank you for your help man, much appreciated.

Comment (lucianosousa) [inline config/initializers/devise.rb]: well done guys!

File: Gemfile [hunk 2]
```diff
@@ -1,14 +1,13 @@
 source 'http://rubygems.org'
 
-ruby '2.1.5'
```
Comment (lucianosousa) [inline Gemfile]: @ricardovaleriano Can I put here the ruby 2.2.0?

Comment (ricardovaleriano) [inline Gemfile]: @lucianosousa Yep, go for it, please.

**Comment by ricardovaleriano** [comment]
Guys, I'll close this PR in favor of #104. There are all the commits from @lucianosousa and some other things that I made, ok?

**Comment by lucianosousa** [comment]
Awesome!

**Pull request closed without merging**